Documentation/coreos.rst

```diff
@@ -0,0 +1,356 @@
+
+* Replace ${ADVERTISE_IP} with this node's publicly routable IP.
+* Replace ${DNS_SERVICE_IP}
+* Replace ${KUBE_VERSION} with a version  ``>= 1.6.4``.
+* Cilium-specific configuration    
+
+    * Mount the CNI configuration directory you created in step 1 so Kubelet can pick up the CNI configuration from the host filesystem:
+
+         ::
+
+             --volume etc-cni,kind=host,source=/etc/cni/net.d \
+             --mount volume=etc-cni,target=/etc/cni/net.d"
+
+    * Mount the directory where CNI plugins are installed:
+
+        ::
+
+             --volume cni-bin,kind=host,source=/opt/cni/bin \
+             --mount volume=cni-bin,target=/opt/cni/bin
+
+    * Mount the BPF filesystem so that the information stored there persists across Cilium restarts:
+
+        ``ExecStartPre=/bin/mount bpffs /sys/fs/bpf -t bpf``
+
+    * Also ensure that you specify that the network plugin is CNI: 
+
+        ``--network-plugin=cni``
+
+    * Specify the CNI directory to correspond to the mount you provided earlier where the CNI configuration is located:
+    
+        ``--cni-conf-dir=/etc/cni/net.d``.
+
+`/etc/systemd/system/kubelet.service`
+:: 
+   [Service]
+   Environment=KUBELET_IMAGE_TAG=v'"${KUBE_VERSION}"'_coreos.0
+   Environment="RKT_RUN_ARGS=--uuid-file-save=/var/run/kubelet-pod.uuid \
+     --volume var-log,kind=host,source=/var/log \
+     --mount volume=var-log,target=/var/log \
+     --volume dns,kind=host,source=/etc/resolv.conf \
+     --mount volume=dns,target=/etc/resolv.conf \
+     --volume cni-bin,kind=host,source=/opt/cni/bin \
+     --mount volume=cni-bin,target=/opt/cni/bin \
+     --volume etc-cni,kind=host,source=/etc/cni/net.d \
+     --mount volume=etc-cni,target=/etc/cni/net.d"
+   ExecStartPre=/usr/bin/mkdir -p /etc/cni/net.d
+   ExecStartPre=/bin/mount bpffs /sys/fs/bpf -t bpf
+   ExecStartPre=/usr/bin/mkdir -p /opt/cni/bin
+   ExecStartPre=/usr/bin/mkdir -p /etc/kubernetes/manifests
+   ExecStartPre=/usr/bin/mkdir -p /var/log/containers
+   ExecStartPre=-/usr/bin/rkt rm --uuid-file=/var/run/kubelet-pod.uuid
+   ExecStart=/usr/lib/coreos/kubelet-wrapper \
+     --api-servers=http://127.0.0.1:8080 \
+     --register-schedulable=false \
+     --cni-conf-dir=/etc/cni/net.d \
+     --network-plugin=cni \
+     --container-runtime=docker \
+     --allow-privileged=true \
+     --pod-manifest-path=/etc/kubernetes/manifests \
+     --hostname-override=${ADVERTISE_IP} \
+     --cluster_dns=${DNS_SERVICE_IP} \
+     --cluster_domain=cluster.local
+   ExecStop=-/usr/bin/rkt stop --uuid-file=/var/run/kubelet-pod.uuid
+   Restart=always
+   RestartSec=10
+
+   [Install]
+   WantedBy=multi-user.target
+
+
+Step 3: Setup kube-apiserver on the master nodes
+--------------------------------
+Follow the `CoreOS instructions for setting up the API server <https://coreos.com/kubernetes/docs/latest/deploy-master.html#set-up-the-kube-apiserver-pod>`_. 
```

Comment:
"Cilium has no special requirements with regards to the `kube-apiserver` setup. Please follow the `CoreOS instructions...`"